app/controllers/doorkeeper/authorizations_controller.rb

```diff
@@ -10,11 +10,14 @@ def new
       else
         render_error
       end
+    rescue Errors::DoorkeeperError => e
```

Comment:
adding `rescue` for `new` and `create` method as at tokens_controller to handle raised error (invalid_request type error): https://github.com/doorkeeper-gem/doorkeeper/blob/1a68d15f9392707dd5e0ab3bb1da900fef3494e6/app/controllers/doorkeeper/tokens_controller.rb#L9